File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SeckillService.java
package com.pinyougou.sellergoods.service;

import java.util.List;

import com.pinyougou.pojo.TbGoods;
import com.pinyougou.pojo.TbSeckillGoods;

import entity.PageResult;

public interface SeckillService {

	List<TbSeckillGoods> findAll();

	void add(TbSeckillGoods tbSeckillGoods);

	PageResult search(TbSeckillGoods tbSeckillGoods, int pageNum, int pageSize);

	TbSeckillGoods findOne(Long id);

	void updateStatus(String status, Long id);

	

	

}

File: pinyougou_parent/pinyougou_shop_web/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	
	this.findAll=function(){
		return $http.get('../seckill/findAll');
	}
	
	this.add=function(entity){
		return  $http.post('../seckill/add',entity );
	}
});
